doc/source/structures/reflection/structure.rst

```diff
@@ -12,10 +12,6 @@ The root of all kOS data types.
 Overview
 --------
 
-.. versionadded:: 0.19.0
-    This is a new feature of kOS v0.19.0.  Most of what is
-    documented on this page won't work on earlier versions.
-
```

Comment:
Agree.  We're not worried about people running new scripts on old versions of kOS, but rather people running old scripts on new versions of kOS.